pkg/occlient/occlient_test.go

```diff
@@ -8,9 +8,74 @@ import (
 	"testing"
 
 	routev1 "github.com/openshift/api/route/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	ktesting "k8s.io/client-go/testing"
 )
 
+func TestCreatePVC(t *testing.T) {
+	tests := []struct {
+		name    string
+		size    string
+		labels  map[string]string
+		wantErr bool
+	}{
+		{
```

Comment:
there should also be some tests that result in error. For example invalid `size`  value